src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1609,7 +1620,8 @@ protected override void InvokeDefaultAction(string path)
             bool recurse,
             uint depth,
             bool nameOnly,
-            ReturnContainers returnContainers)
+            ReturnContainers returnContainers,
+            InodeTracker tracker)   // tracker will be non-null only if the user invoked the -FollowSymLinks switch parameter.
```

Comment:
Minor comment: this comment is not accurate now -- `tracker` will be null when `-FollowSymlinks` is specified but `-Recurse` is not.